src/v/pandaproxy/schema_registry/protobuf.cc

```diff
@@ -141,12 +142,23 @@ class io_error_collector final : public pb::io::ErrorCollector {
     };
 
 public:
+#if PROTOBUF_VERSION < 5027000
```

Comment:
I will seperately upgrade vtools, but the OSS build will have to be nuked before we can fully remove this as far as I can tell

Reply:
I doubt we'll forget about this, but maybe add as a subtask here https://redpandadata.atlassian.net/browse/CORE-5565 so it's not lost?

Reply:
Done